test/unit/test-url-replacements.js

```diff
@@ -1931,6 +1931,21 @@ describes.sandboxed('UrlReplacements', {}, env => {
           );
         });
 
+        it('should concatenate and expand additional params w/ whitelist', () => {
```

Comment:
Could you add another param that does need encoding and make sure we still encode the variable. Thanks.

Reply:
`CLIENT_ID` is not [defined in the `data-amp-replace`](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#appending-parameters-to-the-href) attribute so it's parameters should be encoded.

I added more parameters to `CLIENT_ID` to show that it is encoded

Reply:
I meant  something like `data-amp-addparams=encode,test`